src/main/java/io/openmessaging/storage/dledger/DLedgerServer.java

```diff
@@ -441,19 +441,19 @@ private void checkPreferredLeader() {
         }
     }
 
-    public DLedgerStore getdLedgerStore() {
+    public DLedgerStore getDLedgerStore() {
         return dLedgerStore;
     }
 
-    public DLedgerRpcService getdLedgerRpcService() {
+    public DLedgerRpcService getDLedgerRpcService() {
         return dLedgerRpcService;
     }
 
-    public DLedgerLeaderElector getdLedgerLeaderElector() {
+    public DLedgerLeaderElector getDLedgerLeaderElector() {
         return dLedgerLeaderElector;
     }
 
-    public DLedgerConfig getdLedgerConfig() {
+    public DLedgerConfig getDLedgerConfig() {
         return dLedgerConfig;
     }
```

Comment:
Directly modifying these interface names will cause compatibility problems

Reply:
add @Deprecated annotation for those methods instead of Directly modifying these interface names. and add a new method like getDLedgerConfig to solution compatibility problems.

Reply:
> Directly modifying these interface names will cause compatibility problems

I search found that RocketMQ uses these methods, and it does cause compatibility issues if you change them directly.add @deprecated annotation for those methods instead of Directly modifying these interface names. and add a new method like getDLedgerConfig to solution compatibility problems.

Reply:
> > Directly modifying these interface names will cause compatibility problems
> 
> I search found that RocketMQ uses these methods, and it does cause compatibility issues if you change them directly.add @deprecated annotation for those methods instead of Directly modifying these interface names. and add a new method like getDLedgerConfig to solution compatibility problems.

Good idea！